src/Command/CodeStudio/CodeStudioPipelinesMigrateCommand.php

```diff
@@ -0,0 +1,612 @@
+    $gitlab_cicd_existing_variables = $this->gitLabClient->projects()->variables($project['id']);
+    foreach ($gitlab_cicd_existing_variables as $variable) {
+      if ($variable['key'] == $GLAB_TOKEN_NAME) {
+        return $variable;
+      }
+    }
+    foreach ($gitlab_cicd_existing_variables as $variable) {
+      if ($variable['key'] != $GLAB_TOKEN_NAME) {
+        throw new AcquiaCliException("[Error] Code Studio CI/CD environment 'Variable_Name' variable is not configured properly. \n[Help] To set all the CI/CD environment variables, configure the project in Code Studio by using 'acli cs:wizard' command \n You can find Code Studio documentation at https://docs.acquia.com/code-studio/getting-started/#create-code-studio-project \n OR \n Set the above required CI/CD environment variables in Code Studio project.\n");
+      }
+    }
+  }
+
+  //todo : check the current directry == projects;
+
+  /**
+   * Check acquia-pipeline.yml file exists in the root repo and remove ci_config_path from codestudio project.
+   * @param array $project
+   * @throws \Acquia\Cli\Exception\AcquiaCliException
+   */
+  protected function checkPipelineExists(array $project) {
+    $pipelines_filepath_yml = Path::join($this->repoRoot, 'acquia-pipelines.yml');
+    $pipelines_filepath_yaml = Path::join($this->repoRoot, 'acquia-pipelines.yaml');
+    if ($this->localMachineHelper->getFilesystem()->exists($pipelines_filepath_yml) or $this->localMachineHelper->getFilesystem()->exists($pipelines_filepath_yaml)) {
+      // print("file exists");
+      $this->gitLabClient->projects()
+        ->update($project['id'], ['ci_config_path' => '']);
+      // print_r("\nRemoved ci_config_path \n");
+      foreach (glob("acquia-pipelines.*") as $this->filename) {
+        echo "File name is : $this->filename\n";
+      }
+      $file_contents = file_get_contents($this->filename);
+      $this->acquia_pipeline_file_parse = Yaml::parse($file_contents, Yaml::PARSE_OBJECT);
+    }
+    else {
+      throw new AcquiaCliException("[Error] Missing 'acquia-pipelines.yaml' file which is required to migrate the project to Code Studio.");
+    }
+  }
+
+  /**
+   * Migrating standard template to .gitlab-ci.yml file.
+   */
+  protected function migrateCSStandardTemplate() {
+    $auto_devops_pipeline = [
+        'include' => ['project' => 'acquia/standard-template', 'file' => '/gitlab-ci/Auto-DevOps.acquia.gitlab-ci.yml'],
+      ];
+    $auto_devops_dump_file = Yaml::dump($auto_devops_pipeline);
+    $auto_devops_parse_file = Yaml::parse($auto_devops_dump_file);
+    $this->emptyArray = array_merge($this->emptyArray, $auto_devops_parse_file);
+    $this->createGitlabCiFile($this->emptyArray);
+  }
+
+  //todo : catch execpetion if any and throw error.
+
+  /**
+   * Migrating varibales to .gitlab-ci.yml file.
+   */
+  protected function migrateVariables() {
+    $varData = isset($this->acquia_pipeline_file_parse['variables'])?$this->acquia_pipeline_file_parse['variables']:FALSE;
```

Comment:
This is probably going to fail PHPCS for a few reasons.

Reply:
I tested it PHPCS is passing.